tests/test_vmapoutput.py

```diff
@@ -0,0 +1,118 @@
+from vmap4fenics import VMAP4Fenics
+import PyVMAP as VMAP
+import numpy as np
+import fenics_concrete
+from os.path import isfile
+import pytest
+
+def test_vmapoutput():
+  # define problem
+  parameters = fenics_concrete.Parameters()
+  parameters['E'] = 30000
+  parameters['nu'] = 0.2
+  parameters['height'] = 300
+  parameters['radius'] = 75
+  parameters['mesh_density'] = 6
+  parameters['log_level'] = 'WARNING'
+  parameters['bc_setting'] = 'fixed'
+  parameters['dim'] = 3
+  experiment = fenics_concrete.ConcreteCylinderExperiment(parameters)
+  problem = fenics_concrete.LinearElasticity(experiment = experiment, parameters = parameters, pv_name = 'test', vmapoutput = True)
+  # define sensors
+  problem.add_sensor(fenics_concrete.sensors.ReactionForceSensorBottom())
+  problem.set_material(name = 'Linear_Concrete',
+            state = 'solid',
+            type = 'concrete',
+            description= 'linear elastic model',
+            material_id = 'linear_concrete_model',
+            idealization = 'continuum',
+            physics = 'solid mechanics'
+            )
```

Comment:
You dont have to change this now, but for the future, this looks like information that could be directly included in the fenics_concrete.LinearElasticity class.